source/combat.cpp

```diff
@@ -2117,6 +2117,8 @@ SI16 CHandleCombat::ApplyDamageBonuses( WeatherType damageType, CChar *mChar, CC
 	R32 multiplier = 1;
 	R32 damage = 0;
 	SI32 RaceDamage = 0;
+	// Add Damage Increase (DI) from items, buffs, etc.
+	SI16 damageIncreasePercent = mChar->GetDamageIncrease(); // Total DI from items, buffs, etc.
```

Comment:
This only applies to PHYSICAL damage currently, so move it into the switch section, under case PHYSICAL: (just before the other changes there)